app/presenters/content_item/contents_list.rb

```diff
@@ -3,6 +3,7 @@ module ContentsList
     include ActionView::Helpers::UrlHelper
     include TypographyHelper
 
+    WORD_COUNT_LIMIT = 100
```

Comment:
I wonder if total word count is a useful heuristic.

Reply:
What @fofr said. Some formats will contain medical terminology with superlongtechnicalwords, some will contain more legalese. I think a char count might be a more honest metric.